resource.go

```diff
@@ -15,6 +20,48 @@ const (
 	ResourceTypeDelegationToken ResourceType = 6
 )
 
+func (rt ResourceType) String() string {
+	mapping := map[ResourceType]string{
+		ResourceTypeUnknown:         "Unknown",
+		ResourceTypeAny:             "Any",
+		ResourceTypeTopic:           "Topic",
+		ResourceTypeGroup:           "Group",
```

Comment:
Are you missing ResourceTypeBroker here?

Reply:
`ResourceTypeBroker` and `ResourceTypeCluster` have the same value associated with them and a map cannot have duplicate keys
https://segment.atlassian.net/browse/DP-1858

Reply:
👍 maybe a comment to that effect for future code reviewers